.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Java (${{ matrix.java }}) and Maven
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Perform Maven Release
+        uses: qcastel/github-actions-maven-release@master
+        with:
+          release-branch-name: "master"
+
+          git-release-bot-name: "rtr-actions"
+          git-release-bot-email: "infrastructure+github@renttherunway.com"
+
+          gpg-enabled: "true"
+          gpg-key-id: ${{ secrets.GPG_KEY_ID }}
+          gpg-key: ${{ secrets.GPG_KEY }}
+
+          maven-repo-server-username: ${{ secrets.OSS_NEXUS_USERNAME }}
```

Comment:
Can you make these org-level?

Reply:
I was going to after I verified the workflow was working correctly as I can't add them myself for testing.

Reply:
Actually, I just remembered, you can't make them org-level.  You can't have a public repo use org-level secrets for security reasons :/